.vscode/local_dnstap.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
```

Comment:
Please don't include your local editor files in the repo.

Reply:
sorry, my bad. 

https://github.com/coredns/coredns/pull/6459/commits/78b0f53f9d6fa9510c65fc9a1f2957c509f36757